Documentation/ProjectFile/prj/processes/process/THERMO_HYDRO_MECHANICS/t_specific_body_force.md

```diff
@@ -0,0 +1 @@
+Specific body forces that are used to apply gravitational forces, a vector with a size of the number of displacement components.
```

Comment:
Maybe it is more understandable to write:
... a vector of size equal to the displacement dimension.

Reply:
Corrected.